manifests/a/Autodesk/NavisworksManage2022/2022/Autodesk.NavisworksManage2022.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.9.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Autodesk.NavisworksManage2022

```

```suggestion
PackageIdentifier: Autodesk.NavisworksManage.2022
```

After changing the package identifier, you will also need to change the path of the manifests to `./manifests/a/Autodesk/NavisworksManage/2022/2022/*`